apps/vaporgui/SettingsParams.cpp

```diff
@@ -150,6 +151,7 @@ void SettingsParams::SetCacheMB(long val)
 {
     if (val < 0) val = 8000;
     SetValueLong(_cacheMBTag, "Set cache size", val);
+    SaveSettings();
```

Comment:
SaveSettings() can fail and return an error code. You're ignoring the return value and the user will never see the error message. Is there a reason that SaveSettings has been added to every setter and not simply called once higher up the stack as was done before?

Reply:
SaveSettings is now performed in Update(), and the user gets notified upon a bad error code.